### PR TITLE
Changelog r119

### PR DESCRIPTION
**Pull request by esbenbach**
Added changelog for r119 - a bit on the lite side this time around...

### CODE REVIEW

File: source/changelog/r119.inc [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+* Exposed existing Payment Provider Types for EHF and Email in various workflows.
+* All credit operations now go through the new endpoint and credit flows will transfer back account transactions as mentioned in the previous release.
+* Removed/Sanitized the Invoices and Payments grid in the subscriber overview.
+* Refunds are now highlighted with a separate colour.
+
+Fixed
+-----
+
+* Re-introduced option for quick navigation to Enterprise Plan owners from the Enterprise Plan configuration.
```
Comment (KleisKlasKluss) [inline source/changelog/r119.inc:62]: Has this been a possibility in previous versions as well?

I know we re-introduced the navigation from EP subscribers and subscriptions directly to the subscriber and its subscription in the Subscriptions and Subscribers grid

File: source/changelog/r119.inc [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+===============
+The following new features and improvements are now available in the merchant client.
+
+Added
+-----
+
+* Option to toggle whether to reduce reminder amounts with surplus accounted payments (allowances).
+* It is now possible to cancel Vipps Payment Agreements.
+* When ordering from merchant it is possible to choose payment types besides Invoice.
+* Adding an Organization Number for tenants with Norwegian organizations will now lookup in Elma and provide a hint regarding EHF Invoice validity.
```
Comment (havmunke) [inline source/changelog/r119.inc:49]: This was added with an earlier sprint.
What is new now, is that the organization-number is validated if the user try to switch to an EHF-agreement.

**Review by BrynjarS — APPROVED** [review]

File: source/changelog/r119.inc (hunk 1 above)
Comment (esbenbach) [reply source/changelog/r119.inc:62]: Not sure, that was just my understanding

Comment (esbenbach) [reply source/changelog/r119.inc]: Changed it...

Comment (havmunke) [reply source/changelog/r119.inc]: This is actually new. Story 16072.
What Klas describes, was a fix of Bug 16255

**Review by havmunke — APPROVED** [review]

**Pull request merged**